client/graphics/src/context.rs

```diff
@@ -1,7 +1,19 @@
 use crate::graphics::WebGl;
 use crate::render::Render;
+use lazy_static::lazy_static;
+use rask_engine::resources::*;
```

Comment:
Why do you need this import? If you really need it, it would be cleaner to split it into concrete imports like the line below.